### DISCUSSION TITLE
Dexcom G7 App Update and Reading

### DISCUSSION THREAD

**Discussion by nekromantik** [NightscoutFoundation/xDrip#2696] [General]
Hello

Anyone else have a problem with latest version of g7 app and xdrip companion mode where xdrip looses reading's when you disable dexcom quick glance?

This is on Google Pixel 7 Pro

Thanks

**Comment by Navid200** [comment]
Companion app mode uses notifications from the other app.  If you disable quick glance, you disable notifications.

**Reply by nekromantik** [reply]
> Companion app mode uses notifications from the other app. If you disable quick glance, you disable notifications.

I still get notifications like high or low sugar levels. It disable the graph on notification drawer.

**Reply by Navid200** [reply]
Are you getting notifications only if you are high or low but not if you are in range?

**Reply by nekromantik** [reply]
> > Companion app mode uses notifications from the other app. If you disable quick glance, you disable notifications.
> 
> I still get notifications like high or low sugar levels. It disable the graph on notification drawer.

Even with quick glance turned on I don't get notifications if in range. Only difference is I can see my current level on notification screen without having to open app.

**Reply by Navid200** [reply]
Would you please post a screenshot of the notification screen with quick glance enabled?

**Reply by nekromantik** [reply]
> Would you please post a screenshot of the notification screen with quick glance enabled?

Sure
Two screenshots first with quick glance and second without
![Screenshot_20230223-150142](https://user-images.githubusercontent.com/90213527/220944947-c61c8ad2-e812-42d8-b127-db61d2d969fb.png)
![Screenshot_20230223-150159](https://user-images.githubusercontent.com/90213527/220944995-9c1e03ca-fc50-42e9-b630-30033874e085.png)

**Reply by Navid200** [reply]
It's quite possible we are misunderstanding each other.
This is what xDrip gets its readings from in Companion app mode.
![220944947-c61c8ad2-e812-42d8-b127-db61d2d969fb](https://user-images.githubusercontent.com/51497406/220945908-addf68bf-61d9-48b4-9209-aefc835a79d3.png)

**Comment by jamorham** [comment]
There is also the companion bluetooth feature which can also help with things like back-filling. If the user doesn't want the official app notification visible then often it still works when hidden by long pressing on the notification but this may vary between handsets and android versions so needs testing.

**Comment by nekromantik** [comment]
Okay thanks both.
Guess I will try and hide the notification.
Will wait for g7 to enable broadcast so don't need companion mode
